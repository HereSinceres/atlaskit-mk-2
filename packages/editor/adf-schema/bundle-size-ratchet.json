[
  {
    "id": "src.main",
    "name": "main",
    "stats": {
      "size": 56835,
      "gzipSize": 13863
    }
  },
  {
    "id": "node_modules.main",
    "name": "node_modules [main]",
    "stats": {
      "size": 60668,
      "gzipSize": 18855
    }
  },
  {
    "id": "combined.main",
    "name": "main",
    "stats": {
      "size": 116889,
      "gzipSize": 32050
    }
  }
]