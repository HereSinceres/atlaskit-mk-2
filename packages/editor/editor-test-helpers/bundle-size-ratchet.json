[
  {
    "id": "src.main",
    "name": "main",
    "stats": {
      "size": 41802,
      "gzipSize": 12153
    }
  },
  {
    "id": "node_modules.main",
    "name": "node_modules [main]",
    "stats": {
      "size": 1283257,
      "gzipSize": 362123
    }
  },
  {
    "id": "node_modules.async",
    "name": "node_modules [async]",
    "stats": {
      "size": 1402458,
      "gzipSize": 398860
    }
  },
  {
    "id": "atlaskit.core.main",
    "name": "main",
    "stats": {
      "size": 277232,
      "gzipSize": 57294
    }
  },
  {
    "id": "atlaskit.core.async",
    "name": "async",
    "stats": {
      "size": 169431,
      "gzipSize": 32811
    }
  },
  {
    "id": "atlaskit.editor.main",
    "name": "main",
    "stats": {
      "size": 695964,
      "gzipSize": 184825
    }
  },
  {
    "id": "atlaskit.editor.async",
    "name": "async",
    "stats": {
      "size": 38913,
      "gzipSize": 9073
    }
  },
  {
    "id": "atlaskit.elements.main",
    "name": "main",
    "stats": {
      "size": 81993,
      "gzipSize": 20568
    }
  },
  {
    "id": "atlaskit.elements.async",
    "name": "async",
    "stats": {
      "size": 71992,
      "gzipSize": 21494
    }
  },
  {
    "id": "atlaskit.media.main",
    "name": "main",
    "stats": {
      "size": 185744,
      "gzipSize": 43792
    }
  },
  {
    "id": "atlaskit.media.async",
    "name": "async",
    "stats": {
      "size": 926651,
      "gzipSize": 228594
    }
  },
  {
    "id": "atlaskit.teamwork-ecosystem.main",
    "name": "main",
    "stats": {
      "size": 624,
      "gzipSize": 367
    }
  },
  {
    "id": "combined.main",
    "name": "main",
    "stats": {
      "size": 2556572,
      "gzipSize": 691279
    }
  },
  {
    "id": "combined.async",
    "name": "async",
    "stats": {
      "size": 2608384,
      "gzipSize": 691878
    }
  }
]