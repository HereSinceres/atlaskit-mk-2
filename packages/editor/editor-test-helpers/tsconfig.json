{
  "extends": "../../../tsconfig.entry-points.json",
  "compilerOptions": {
    "baseUrl": "./"
  },
  "include": [
    "./src/**/*.ts",
    "./src/**/*.tsx",
    "./tests/**/*.ts",
    "./tests/**/*.tsx",
  ]
}