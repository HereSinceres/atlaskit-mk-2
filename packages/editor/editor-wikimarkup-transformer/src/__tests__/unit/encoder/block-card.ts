import { defaultSchema } from '@atlaskit/adf-schema';
import WikiMarkupTransformer from '../../../index';

import { doc, blockCard } from '@atlaskit/editor-test-helpers';

describe('ADF => WikiMarkup - BlockCard', () => {
  const transformer = new WikiMarkupTransformer();

  test('should convert blockcard node', () => {
    const node = doc(blockCard({ url: 'https://www.dropbox.com' })())(
      defaultSchema,
    );
    expect(transformer.encode(node)).toMatchSnapshot();
  });

  test('should convert blockcard node with icft', () => {
    const node = doc(
      blockCard({
        url: 'https://product-fabric.atlassian.net/browse/EX-522#icft=EX-522',
      })(),
    )(defaultSchema);
    expect(transformer.encode(node)).toMatchSnapshot();
  });
});
