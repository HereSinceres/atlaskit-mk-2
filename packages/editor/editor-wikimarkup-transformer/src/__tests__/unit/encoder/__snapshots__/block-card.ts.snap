// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ADF => WikiMarkup - BlockCard should convert blockcard node 1`] = `"[https://www.dropbox.com|https://www.dropbox.com|smart-link]"`;

exports[`ADF => WikiMarkup - BlockCard should convert blockcard node in icft 1`] = `"[EX-522]"`;
